src/content/third_party_migration/vendors/redhat-json.json

```diff
@@ -0,0 +1,30 @@
+{
+    "vendor": "redhat-json",
+    "description": "RedHat JSON",
+    "remotes": [
+        {
+            "url": "https://access.redhat.com/labs/securitydataapi/cve.json",
+            "request": "api",
+            "type": "json",
+            "parameters": {
+                "after": {
+                    "type": "fixed",
+                    "description": "Year",
+                    "value": "1999"
```

Comment:
- after parameter expects ISO 8601 format -> YYYY-MM-DD or YYYYMMDD

Reply:
Set just year on configurations and do the work on the module? I think this will bring lots of if-else if-else statements on the fetcher code